### ISSUE TITLE
Harvester.create raises InvalidParameterException (flir Blackfly camera)

### ISSUE THREAD

**Issue by oliver-batchelor** [genicam/harvesters#408]
**Describe the Issue**

I'm trying harvesters with our cameras and  `create` on any of the devices raises an InvalidParameterException on one of the register_event calls. Any tips on why this doesn't work would be appreciated!

I'm on Ubuntu 20.04, Flir spinnaker SDK `spinnaker-3.0.0.118-amd64`. The cameras work with the usual PySpin API and they show up as GenICam devices, output of script is shown below. 

Just installed harvesters from github. Python 3.8.

```
from harvesters.core import Harvester
h = Harvester()
h.add_file('/opt/spinnaker/lib/flir-gentl/FLIR_GenTL.cti')
h.update()
print(h.device_info_list)
ia = h.create(h.device_info_list[0])
```

Output from running this:
```
[{'access_status': 1, 'display_name': 'FLIR', 'id_': 'USB\\VID_1E10&SRL_0132380C\\&2&9&5', 'model': 'Blackfly S BFS-U3-120S4C', 'parent': <genicam.gentl.Interface; proxy of <Swig Object of type 'std::shared_ptr< GenTLCpp::TLInterface > *' at 0x7f9ea3c95450> >, 'serial_number': '20068364', 'tl_type': 'U3V', 'user_defined_name': '', 'vendor': 'FLIR', 'version': '1801.0.116.0'}, {'access_status': 1, 'display_name': 'FLIR', 'id_': 'USB\\VID_1E10&SRL_012862BD\\&2&8&4', 'model': 'Blackfly S BFS-U3-120S4C', 'parent': <genicam.gentl.Interface; proxy of <Swig Object of type 'std::shared_ptr< GenTLCpp::TLInterface > *' at 0x7f9ea3c95510> >, 'serial_number': '19423933', 'tl_type': 'U3V', 'user_defined_name': '', 'vendor': 'FLIR', 'version': '1801.0.116.0'}]
Traceback (most recent call last):
  File "/home/oliver/Downloads/test_harvester.py", line 15, in <module>
    ia = h.create(h.device_info_list[0])
  File "/home/oliver/Downloads/harvesters/src/harvesters/core.py", line 3082, in create
    return self._create_acquirer(device_proxy=device_proxy, config=config)
  File "/home/oliver/Downloads/harvesters/src/harvesters/core.py", line 3112, in _create_acquirer
    ia = ImageAcquirer(device_proxy=device_proxy_, config=config,
  File "/home/oliver/Downloads/harvesters/src/harvesters/core.py", line 1665, in __init__
    manager(module.register_event(event_type))
  File "/home/oliver/Downloads/harvesters/src/harvesters/core.py", line 228, in m
    return getattr(self._source_object, attribute)(*args)
  File "/home/oliver/mambaforge/envs/conan_ros/lib/python3.8/site-packages/genicam/gentl.py", line 2582, in register_event
    return _gentl.System_register_event(self, event_id)
_gentl.InvalidParameterException: GenTL exception: Parameter not valid or out of range. (Message from the source: Error calling GenTL function system GCRegisterEvent.) (ID: -1009)

```

**Comment by acfarrell** [comment]
I'm getting the same exception when trying to connect to my FLIR Grasshopper3 on Windows 11, Spinnaker 3.1.0.79_x64, Python 3.8. The camera information shows up correctly on h.device_info_list, but h.create(0) throws the same InvalidParameterException.

**Comment by oliver-batchelor** [comment]
Some further digging reveals it throws InvalidParameter for all bar `self._remote_device`  from the list below:

```
        modules = [self._system_proxy,
                   self._interface_proxy,
                   self._device_proxy,
                   self._remote_device]
```

**Comment by Zyl-000** [comment]
having the same question using spinnaker 3.1.0.79 with blackfly S bfs-pge-16s2c cameras on windows

**Comment by sunavlis** [comment]
This issue is solved in release 1.4.3 now available on PyPI.

@Zyl-000 I wasn't able to test the new release with all cameras and producers described in this issue (because I don't have these) but would expect that it is solved as well for the Spinnaker. Feel free to reopen the issue or create a new one, if you still observe the issue on that device.

Thanks for reporting the issue!
